src/examples/mnist/MNIST_SP.cpp

```diff
@@ -88,7 +88,7 @@ void setup() {
     /* synPermConnected */            0.5f, //no difference, let's leave at 0.5 in the middle
     /* minPctOverlapDutyCycles */     0.2f, //speed of re-learning?
     /* dutyCyclePeriod */             1402,
-    /* boostStrength */               2.0f, // Boosting does help, but entropy is high, on MNIST it does not matter, for learning with TM prefer boosting off (=0.0), or "neutral"=1.0
+    /* boostStrength */               0.0f, // Boosting does help, but entropy is high, on MNIST it does not matter, for learning with TM prefer boosting off (=0.0), or "neutral"=1.0
```

Comment:
If `boostStrength == 0` then boosting is disabled altogether, so I do not think your comparison between exponential and logarithmic boosting is valid.

Reply:
good call!! Something go messed up, thanks!1 I'll be updating the results